pkg/agent/openflow/pipeline.go

```diff
@@ -2235,6 +2245,10 @@ func (c *client) generatePipeline() {
 		c.pipeline[endpointDNATTable] = bridge.CreateTable(endpointDNATTable, c.egressEntryTable, binding.TableMissActionNext)
 		c.pipeline[conntrackCommitTable] = bridge.CreateTable(conntrackCommitTable, hairpinSNATTable, binding.TableMissActionNext)
 		c.pipeline[hairpinSNATTable] = bridge.CreateTable(hairpinSNATTable, L2ForwardingOutTable, binding.TableMissActionNext)
+		if runtime.IsWindowsPlatform() {
+			c.pipeline[serviceConntrackTable] = bridge.CreateTable(serviceConntrackTable, conntrackTable, binding.TableMissActionNext)
```

Comment:
Which table do you want to be the previous table of serviceConntrackTable? Or you want to jump to "serviceConntrackTable" only in fow actions but not to make serviceConntrackTable to be a default next table of any other table?

Reply:
Let `serviceHairpinTable` be the previous table.